homebrew-casks/readme.md

```diff
@@ -1,2 +1 @@
-This extension to homebrew allows us to install commerical programs like
-chrome and sublime.
+This extension to homebrew allows us to install commercial programs like chrome and sublime.
```

Comment:
Cool Kids keep text less than ~80 characters a line.  This allows for running commands like `cat readme.md` to show up without truncation on the command line.  You do not need to adhere to this standard for your readmes but that was probably the logic on that line break